CHANGELOG.md

```diff
@@ -31,6 +31,16 @@ Only use disabled form controls if research shows it makes the user interface ea
 
 This was added in [pull request #3187: Add disabled styles for form controls](https://github.com/alphagov/govuk-frontend/pull/3187).
 
+#### Stop the Accordion from remembering and restoring sessions
+
+By default, the [Accordion](https://design-system.service.gov.uk/components/accordion/) will remember which sections a user has expanded and collapsed upon leaving a page. If the user later returns to the page, this layout will be restored, overriding any sections that have been manually set as `expanded` in code.
+
+You can now disable this functionality by using the `rememberExpanded` option in the `govukAccordion` Nunjucks macro.
```

Comment:
Do we need to add a link to the relevant nunjucks macro?

Reply:
I don't think so. The user-facing version of the Nunjucks macro code is already part of the design system page linked in the previous paragraph.

Reply:
ok great